data/a-a.ts

```diff
@@ -0,0 +1,32 @@
+// TODO
+const TEST_CFR = {};
+
+export default {
+  name: "A/A test",
+  description: "Use this to test audiences or to diagnose issues in the system.",
```

Comment:
Ana wrote some copy for the hard coded description for the auto-experimenter-bugzilla ticket, maybe we should consolidate the descriptions?

https://docs.google.com/document/d/1YLgQU-pYsOmZiGXFWCr0CtxxoRMfYXqpkcSxqZs0PwQ/edit?usp=sharing